spec/components/uicontainer.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { MockHelper, TestingPlayerAPI } from '../helper/MockHelper';
+import { UIContainer } from '../../src/ts/components/uicontainer';
+
+let playerMock: TestingPlayerAPI;
+
+describe('UIContainer', () => {
+  beforeEach(() => {
+    playerMock = MockHelper.getPlayerMock();
+  });
+
+  describe('release', () => {
+    it('checks if userInteractionEvents and uiHideTimeout is undefined', () => {
+      const uiContainer = new UIContainer({
+        hideDelay: -1, // With an hideDelay of -1 the uiHideTimeout and userInteractionEvents never will be initialized
+        components: [],
+      });
+
+      uiContainer.release();
+      // Ensure that we don't try to call something on undefined properties
+      expect((uiContainer as any).uiHideTimeout).toBeUndefined();
```

Comment:
How does this check ensure that no calls are made?

Reply:
It actually doesn't but it would crash if so ... I wasn't sure how to test as they are private properties and I don't like tests called `should not crash ...` and without actual expectation.